packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -506,6 +506,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
                           className: Dropdown.slotClassNames.clearIndicator,
                           styles: styles.clearIndicator,
                           accessibility: indicatorBehavior,
+                          ...(!search && { tabIndex: 0 }),
```

Comment:
I think we should add some user-defined label, since the clearIndicator would be tabbable, but it won't have any aria-label to indicate it's purpose to screen-reader users

Reply:
I'd suggest to create a separate behavior for clearIndicator